src/views/InteractionDesigner.vue

```diff
@@ -264,6 +268,7 @@ export default {
         invoke(blockClass, 'install', this)
         Vue.component(`Flow${typeWithoutSeparators}`, uiComponent)
       })
+      return Promise.all(blockInstallers)
```

Comment:
I think what I thought yesterday is basically correct - you can't wait for an async function within a forEach. See here:

https://advancedweb.hu/how-to-use-async-functions-with-array-foreach-in-javascript/

But you're right, we can have an await here instead of then so I've done that!